Documentation/forms-intro.html

```diff
@@ -545,6 +545,48 @@ <h4 class="class">Options</h4>
       </tr>
     </table>
 
+    <h4 class="class" id="Autocomplete">Autocomplete</h4>
```

Comment:
**thought**: Would be cool to see the autocomplete in action in a storybook story as well.

Reply:
I have tests now for the autocomplete

Reply:
they tell stories..